documentation/docs/20-core-concepts/60-state-management.md

```diff
@@ -0,0 +1,94 @@
+---
+title: State Management
+---
+
+Managing state is one of the hardest parts in application development. This section covers various use cases with regards to state management and what to watch out for.
+
+## Careful with global state
```

Comment:
```suggestion
## Avoid global state in SSR
```

Reply:
Is "avoid" a too strong word, given that it's ok to have global state if you make sure to not update it to request-specific values during ssr?

Reply:
I tend think it'd be okay as we can go into greater detail in the text and avoid just means "try not to use" and not "do not use at all"